freezeyt/util.py

```diff
@@ -103,11 +106,19 @@ def import_variable_from_module(
     if not sep:
         if default_variable_name is not None:
             variable_name = default_variable_name
-        else:
+
+        elif default_module_name is not None:
             module_name, variable_name = default_module_name, module_name
 
-    if not variable_name or not module_name:
-        raise ValueError(f'Missing variable or module name: {name!r}')
+    if default_module_name and default_variable_name:
+        raise UsageError("Both default values can not be used simultaneously")
```

Comment:
First you check with `if ... is not None:`, then just with `if ...:`. Since the checks are for the same thing, please be consistent.
(I don't care if an empty string in `default_*_name` is treated as missing or as a bad variable/module name – either is reasonable. Butit should be one or the other.)

As an aside, consider this pattern. It's another way to do it. I'm used to it, and it minimizes the number of comparisons, but I'm not sure if it's more readable (which is what matters here).
```python
if default_variable_name is not None:
    if default_module_name is not None:
        raise UsageError("Both default values can not be used simultaneously")
    variable_name = default_variable_name
elif default_module_name is not None:
    module_name, variable_name = default_module_name, module_name
```

Reply:
I supposed that we could get to vars `module_name` and `variable_name` empty string because of using `partition`. So comparison with `None` won't be really exact. I considered that vars as `default_*_name` are different group of var so I can split type of comparison for these.

Reply:
I was talking about `default_*_name`, in the original code:
`if default_variable_name is not None:`
vs.
`if default_module_name and default_variable_name:`

But now the issue isn't there any more.